scripts/run-front-playwright-container.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -e
+
+if [[ $# -gt 1 || ( -n "$1" && "$1" != "--ui-x11" && "$1" != "--ui-wayland" ) ]]; then
+    echo "Usage is run-front-playwright, run-front-playwright --ui-x11, or run-front-playwright --ui-wayland"
+    exit 1
+fi
+
+# Open the base osrd folder, assuming the script is located in osrd/scripts
+cd "$(dirname "$0")/.." 
+
+docker build -t osrd-dev-playwright:latest -f front/docker/Dockerfile.playwright .
+
+# Switch between the x11 or wayland ui mode or the headless mode
+if [[ "$1" == "--ui-x11" ]]; then
+    xhost +local:docker
```

Comment:
I'm not sure why this is required, and I'm pretty sure it's wrong anyway.

Quoting from xhost manpage:

> The local family specifies all the local connections at once. However, the server interpreted address "si:localuser:username" can be used to specify a single local user. (See the Xsecurity(7) manual page for more details.)

So specifying docker here actually has no effects, all local users are now allowed access to the X server if I understand this correctly (@emersion ?) 

But anyway, isn't the user using the socket inside the container root anyway?

Reply:
It didn't work for me without 

But anyway I will use emersion's solution which does not require this

Reply:
Deleted code. Thanks though!